tools/readme_generator/make_readme.py

```diff
@@ -106,7 +106,7 @@ def generate_READMEs(app_path: Path):
                 continue
             screenshots.append(str(entry.relative_to(app_path)))
 
-    def generate_single_README(lang_suffix: str, lang: str):
+    def generate_single_README(lang_suffix: str, lang: str, translation_warning: bool = False):
```

Comment:
Why do you pass "translation_warning" as a function parameter? Reading the code it looks like this should simply be a variable in the function '-'

Reply:
Ok, I got it reading the rest of the code ^w^

Reply:
Let me detail this for future reference; translation_warning is: 
- passed to `generate_single_README()` as parameter when the warning relates to the weblate translation (readme structure)
- defined within `generate_single_README()` when the warning relates to the github translation (description.md and disclaimer.md). Not sure though if you will want to keep this behavior - cf. the 3rd checkbox in my message hereinabove.